packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -694,8 +684,6 @@ function completeWork(
       break;
     }
     case ForwardRef:
-    case ForwardRefLazy:
-      break;
```

Comment:
Oops, shouldn't have deleted the break, because now it falls through to `SuspenseComponent`. I'll revert.